tests/Rules/UnusedViewsRuleTest.php

```diff
@@ -42,14 +42,6 @@ protected function getCollectors(): array
         ];
     }
 
-    protected function setUp(): void
-    {
-        parent::setUp();
-
-        // This is a workaround for a weird PHPStan container cache issue.
-        require __DIR__ . '/../../bootstrap.php';
-    }
-
```

Comment:
Why is it removed?

Reply:
I allow both and there's no reason not to support it---it's going to have to be done eventually.

The fact also remains that this "weird PHPStan container cache issue" no longer seems to exist given that the tests are passing just fine